css-logical-props/Overview.bs

```diff
@@ -10,7 +10,7 @@ ED: https://drafts.csswg.org/css-logical-props/
 Editor: Rossen Atanassov, Microsoft, ratan@microsoft.com
 Editor: Elika J. Etemad / fantasai, Invited Expert, http://fantasai.inkedblade.net/contact
 Abstract: This module introduces logical properties and values that provide the author with the ability to control layout through logical, rather than physical, direction and dimension mappings. The module defines logical properties and values for the features defined in [CSS21]. These properties are writing-mode relative equivalents of their corresponding physical properties.
-Ignored Terms: div, dl, dfn
+Ignored Terms: div, dl, dfn, block-start, block-end, inline-start, inline-end, start, end
```

Comment:
Ok, I've addressed this and the next comment in https://github.com/w3c/csswg-drafts/pull/212/commits/4447eecd147b717e522e717e7aa9dfe7ddac76d6.

> If the definitions are broken that's something to be fixed, not ignored.

My thought was that they are already defined in the prose. Those values affect many different properties in several specs, so I wonder how that can be defined correctly.

Sebastian